File: Examples/DotNET/SDK/CSharp/Section/Header-and-Footer/RemoveHeaderAndFooter.cs
using System;
using Aspose.Cloud;

namespace CSharp.Section.Header_and_Footer
{
    class RemoveHeaderAndFooter
    {
        public static void Run()
        {
            string inputfile = "header-footer.doc";
            string subdirection = "Section";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirection);

            // Remove all Headers and Footers
            Common.WordsService.WordsHeadersFooters.DeleteDocumentHeadersAndFooters(inputfile, string.Empty, Common.GetOutputFilePath(inputfile,true), Common.FOLDER);

            // Download output file from cloud storage and save on local directory
            string dataDir = Common.DownloadFile(inputfile, subdirection);

            Console.WriteLine("\nAll headers and footers removed successfully.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/DotNET/SDK/VisualBasic/Section/Header-and-Footer/RemoveHeaderAndFooter.vb

Imports Aspose.Cloud

Namespace Section.Header_and_Footer
    Class RemoveHeaderAndFooter
        Public Shared Sub Run()
            Dim inputfile As String = "header-footer.doc"
            Dim subdirection As String = "Section"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirection)

            ' Remove all Headers and Footers
            Common.WordsService.WordsHeadersFooters.DeleteDocumentHeadersAndFooters(inputfile, String.Empty, Common.GetOutputFilePath(inputfile, True), Common.FOLDER)

            ' Download output file from cloud storage and save on local directory
            Dim dataDir As String = Common.DownloadFile(inputfile, subdirection)

            Console.WriteLine(Convert.ToString(vbLf & "All headers and footers removed successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace
